cmd/monaco/integrationtest/v2/test-resources/integration-all-configs/project/openpipeline/events.json

```diff
@@ -0,0 +1,60 @@
+{
+  "customBasePath": "/platform/ingest/custom/events",
+  "editable": true,
+  "endpoints": [
+    {
+      "basePath": "/platform/ingest/v1/events",
+      "builtin": true,
+      "defaultBucket": null,
+      "displayName": "Default API",
+      "editable": false,
+      "enabled": true,
+      "processors": [],
+      "routing": {
+        "type": "dynamic"
+      },
+      "segment": ""
+    }
+  ],
+  "id": "events",
+  "pipelines": [
+    {
+      "builtin": true,
+      "dataExtraction": {
+        "editable": false,
+        "processors": []
+      },
+      "displayName": "events",
+      "editable": false,
+      "enabled": true,
+      "id": "default",
+      "metricExtraction": {
+        "editable": false,
+        "processors": []
+      },
+      "processing": {
+        "editable": false,
+        "processors": []
+      },
+      "securityContext": {
+        "editable": false,
+        "processors": []
+      },
+      "storage": {
+        "catchAllBucketName": "default_events",
+        "editable": false,
+        "processors": []
+      },
+      "type": "default"
+    }
+  ],
+  "routing": {
+    "catchAllPipeline": {
+      "editable": false,
+      "pipelineId": "default"
+    },
+    "editable": true,
+    "entries": []
+  },
+  "version": "1718793081420-294062f9e129495899197a5c9a0fb83a"
```

Comment:
q: can the version be stripped from the sample? IIRC it's ignored on create/update anyway and generated by the API?

Reply:
yep